core/build.gradle

```diff
@@ -68,6 +68,7 @@ dependencies {
     api ('org.rnorth.duct-tape:duct-tape:1.0.8') {
         exclude(group: 'org.jetbrains', module: 'annotations')
     }
+    api 'com.google.cloud.tools:jib-core:0.22.0'
```

Comment:
btw this should be provided, we don't want to have Jib as a mandatory dependency